### DISCUSSION TITLE
Jittery Video

### DISCUSSION THREAD

**Discussion by blkkns4845** [dgreif/ring#1044] [Q&A] [answered]
Hi,

I am running the plugin on a Raspberry Pi 4 (hardwired to a router). My Ring video is jittery through the Home(kit) app. It freezes every few seconds. I have read as much as I can but didn't see anything to help resolve this. I switched the plugin settings so the cameras are unbridged but didn't see any improvements.

Any ideas?

Thanks.

**Comment by dgreif** [comment, marked as answer]
The plugin simply passes video packets through and does not transcode them at all. Unfortunately this is the best experience I can provide, so I don't have any good advice on how to improve the quality
